server/routes/barcode/newContactValidator.test.ts

```diff
@@ -83,4 +83,15 @@ describe('newContactValidator', () => {
       ])
     })
   })
+  it(`should not allow names longer than 60 characters`, () => {
```

Comment:
I'm OK with this, as I'm sure any sensible pragmatic person would be, but I did notice KH squirming in the standup when you showed it. You just know he's going to say "you can't exclude people with really long names" !

Reply:
When he finds a name longer than 60 chars I'll listen, but pragmatism must prevail if we ever want to put this live. (Sorry I know I'm preaching to the choir!)